How K8s docs handle third party and dual sourced content

zacharysarah opened the pull request:
This PR adds a blog post about how Kubernetes docs handle third party and dual-sourced content.

Blog reviewers: 👋 Hi there! Just because I'm a docs chair and a professional editor doesn't mean this post is in any shape to publish.

Please edit helpfully, mercilessly, and without deference. 🔍 🖊️ 🩸 

/area blog
/assign @kbarnard10 @onlydole

justaugustus commented:
/cc
Would like to review, but don't block on me :)

cblecker commented on content/en/blog/_posts/2020-05-xx-third-party-dual-sourced-content.md:
```diff
@@ -0,0 +1,63 @@
+
+### Re-homing content
+
+Some content will be removed that readers may find helpful. To make sure readers have continous access to information, we're giving stakeholders untilthe [1.19 release deadline for docs](https://github.com/kubernetes/sig-release/tree/master/releases/release-1.19), **July 9th, 2020** to re-home any content slated for removal.
```
```suggestion
Some content will be removed that readers may find helpful. To make sure readers have continous access to information, we're giving stakeholders until the [1.19 release deadline for docs](https://github.com/kubernetes/sig-release/tree/master/releases/release-1.19), **July 9th, 2020** to re-home any content slated for removal.
```

zacharysarah replied:
> Should "third party" actually be "third-party"

Either way is valid in this instance.

> Either way, it's used in both its forms

...as long as it's consistent. 🤦 Good catch!

mrbobbytables commented:
/cc

sftim commented on content/en/blog/_posts/2020-05-xx-third-party-dual-sourced-content.md:
```diff
@@ -0,0 +1,63 @@
+url: /blog/2020/05/third-party-dual-sourced-content
+---
+
+**Author:** Zach Corleissen | [GitHub](https://github.com/zacharysarah/) | [Twitter](https://twitter.com/zachorsarah)
```
I expected to see:
```suggestion
**Author:** Zach Corleissen, Cloud Native Computinf Foundation
```

zacharysarah replied:
I'll incorporate the idea but not the spelling error. 😆

sftim replied:
D'oh!

sftim commented:
Looks fine to me @zacharysarah

kbhawkey commented on content/en/blog/_posts/2020-05-xx-third-party-dual-sourced-content.md:
```diff
@@ -0,0 +1,63 @@
+> Docs can link to third-party open source software (OSS) outside the Kubernetes project if it’s necessary for Kubernetes to function.
+
+These common-sense guidelines make sure that Kubernetes docs document Kubernetes.
+
```
nit: common sense?
nit: These common sense guidelines make sure that Kubernetes docs [fairly|wholly|fully] document Kubernetes.

kbhawkey commented on content/en/blog/_posts/2020-05-xx-third-party-dual-sourced-content.md:
```diff
@@ -0,0 +1,63 @@
+Our goal is for Kubernetes docs to be a trustworthy guide to Kubernetes features. To achieve this goal, SIG Docs is [tracking third party content](https://github.com/kubernetes/website/issues/20232) and removing any third party content that isn't: 
+
+- in-project; **and**
+- required for Kubernetes to function.
```
nit: This list construction may be confusing. Remove if (instead of isn't)?
outside of project
in project or outside of project, but not required for Kubernetes ...

zacharysarah replied:
@kbhawkey The condition is `in-project && required-to-function`, so the formulation is correct. I do agree that the construction is more confusing than prose.

kbhawkey commented on content/en/blog/_posts/2020-05-xx-third-party-dual-sourced-content.md:
```diff
@@ -0,0 +1,63 @@
+
+## Ways to contribute
+
+We're tracking third-party content in an [issue in the website repository](https://github.com/kubernetes/website/issues/20232). If you see third party content that's out of project and isn't required for Kubernetes to function, please comment on the tracking issue. 
```
nit: ... an issue in the Kubernetes website repository?

kbhawkey replied:
Any thoughts about moving the `Ways to contribute` after the `dual-sourced` paragraph?
The `contribute` and `want to know more` seem to flow together.

zacharysarah replied:
> Any thoughts about moving the Ways to contribute after the dual-sourced paragraph?

Good idea!

zacharysarah commented:
@kbarnard10 @onlydole I've incorporated review feedback and squashed commits. If everything looks good to you, let me know your preferred publishing date and I'll update the file name and slug. 👍

onlydole commented:
This looks great to me, @zacharysarah! I know there is another blog post dropping tomorrow, but Wednesday is free (2020-05-06) if that sounds good to you?

zacharysarah commented:
@onlydole That works for me. File/date updated! ✅

onlydole commented:
Thank you, @zacharysarah! 

/lgtm
/approve
